CHANGELOG.md

```diff
@@ -15,6 +16,43 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent release-candidate(s), only the delta to the previous release-candidate
 * for a public release, the consolidated changes since the previous public release
 
+# r1.3
+
+## Release Notes
+
+This release contains the definition and documentation of:
+* energy-footprint-notification v0.1.0
+
+The API definition(s) are based on
+* Commonalities v0.6.0
+* Identity and Consent Management v0.4.0
+
+## energy-footprint-notification v0.1.0
+This is the first public release of the CAMARA Energy Footprint Notification API supporting the retrieval of energy consumption information and carbon footprint for a Service running over some application instances typically edge located.
+
+- API definition **with inline documentation**:
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/EnergyFootprintNotification/blob/r1.3/code/API_definitions/energy-footprint-notification.yaml)
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/EnergyFootprintNotification/r1.3/code/API_definitions/energy-footprint-notification.yaml&nocors)
+  - [View it on Swagger Editor](https://camaraproject.github.io/swagger-ui/?url=https://raw.githubusercontent.com/camaraproject/EnergyFootprintNotification/r1.3/code/API_definitions/energy-footprint-notification.yaml)
+
+### Added
+* Support for the following use cases: https://github.com/camaraproject/EnergyFootprintNotification/discussions/11
+  
+### Changed
+* Renamed operations: https://github.com/camaraproject/EnergyFootprintNotification/pull/83 
+* Changed OAuth2 with OpenId: https://github.com/camaraproject/EnergyFootprintNotification/pull/83
+* Updated the name of the test file to be coherent with the operationId: https://github.com/camaraproject/EnergyFootprintNotification/pull/83
+* info.description reordering: https://github.com/camaraproject/EnergyFootprintNotification/pull/92
+* ErrorInfo update according to Commonalities 6.0 public release: https://github.com/camaraproject/EnergyFootprintNotification/pull/95
+
+### Fixed
+* operationId calculatEnergyConsumption: https://github.com/camaraproject/EnergyFootprintNotification/issues/88
```

Comment:
As this is the changelog for the public release actually everything is new, but also fine to list the latest changes. But would make sense to add a line to which pre-release you are comparing.